### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by shlok2740**
## 🛠️ Fixes Issue
Added some blockchains and marketplaces
Deleted repeated resources like SDK and oracles
Corrected the index

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -398,37 +401,6 @@ Decentralized Storage [thread](https://twitter.com/FrancescoCiull4/status/148329
 - [Metaschool](https://metaschool.so/) Metaschool is a platform where developers hangout to learn web3 with each other, collaborate on projects, earn NFTs for completing their work, and connect with talented folks building the future of the web!
 
 
-<div align="right">
```
Comment (FrancescoXX) [inline README.md:401]: why did you remove this part?

Comment (shlok2740) [reply README.md:401]: Because there will be two of this tag

Comment (shlok2740) [reply README.md:401]: Check at line 434

File: README.md [hunk 2]
```diff
@@ -398,37 +401,6 @@ Decentralized Storage [thread](https://twitter.com/FrancescoCiull4/status/148329
...
-</div>
-
-## Off-Chain Data Protocols
-
-- [Ceramic Network](https://ceramic.network/) Ceramic is a decentralized data network that brings unlimited data composability to Web3 applications.
-- [Gun](https://gun.eco/) The decentralized distributed graph database engine. GunDB is the Open Source Firebase alternative, a web3 data protocol.
-
 <div align="right">
   <b><a href="#index">↥ Back To Top</a></b>
 </div>
```
Comment (shlok2740) [inline README.md:406]: I kept this one
